### PR TITLE
Enables upgrading and supercooling servers to produce more points

### PR DESCRIPTION
**Pull request by xTrainx**
:cl: xTrainx
tweak: Servers will now produce 33% more points per tier of parts, double with triphasic scanners. They also linearly produce more points the colder they are, double at the lowest possible temperature. This can stack to a maximum of 4x point generate rate. The default, round-start point generation has not been changed.
/:cl:

[why]: #Original pr by Buggy123 for TGstation #33719, just copied from that, was here before but needed to be modularized

### CODE REVIEW

**Comment by dreamcrab** [comment]
wat

**Comment by ghost** [comment]
in layman's terms, what the fuck does any of this mean

**Comment by JohnGinnane** [comment]
@xTrainx would you mind linking original PR and just fixing the formatting in your changelog? The "balance" section looks to be part of the first message

**Comment by praisenarsie** [comment]
a wise man once said:

firefox_2018-01-06_00-44-07

@Dudeinyourbackyard 
if you upgrade the rnd servers and make a spacetile in the rnd server room, it makes rnd ridiculously op

**Comment by ghost** [comment]
then no -1

**Comment by GuyonBroadway** [comment]
He says op but R&D got quadruple nerfed with techwebs. Not only can you not make verything yoy research, not only is it slower to research shit, not only can you not produce the machines to make all your shit but other department can spend your precious R&D points on usless crap and have even LESS incentive to deliver minerals to you.

**Comment by JohnGinnane** [comment]
The original PR was closed 🤔

**Comment by ghost** [comment]
eww no

**Comment by praisenarsie** [comment]
@GuyonBroadway read everyone's comments in the original /tg/ pr (especially goof).

**Comment by WarRage333** [comment]
Takes like 30 minutes (pulling out of my ass may changed since new 3k~ point start generation) to make a mech, when a avarage hippie round lasts 20 minutes lmao.

Wish we had ways to increase the point gain

**Comment by JohnGinnane** [comment]
\>hippe round lasts 20 minutes lmao

https://tools.hippiestation.com/stats/stats/monthlyStats.php?year=2017&month=11

Not even two full months ago

**Comment by JohnMcLaime** [comment]
I am kinda against this too, it would just make it a must and extra tedious work if you wanted to do rnd, maybe find another way which requires you to risk something else than time.

**Comment by JohnGinnane** [comment]
I think destructive analyser could be given function that lets you gain reserach points from exotic/illegal items, but either only once or with diminishing returns

**Comment by JohnMcLaime** [comment]
ATM destructive analyser is pretty much useless, i think you can only get ayylmao tech (which is overpriced for the crap it gives, to begin with and you can only open 1 ayylium node per tool you put in the analyser) and material reclamation is rarely needed.

**Comment by JohnGinnane** [comment]
All the more reason to "buff" it so at least gives you something, e.g. scanning tator loot for sweet points

**Comment by praisenarsie** [comment]
bro DA boosts some areas of tech lol

**Comment by JohnMcLaime** [comment]
All i have seen it boost was pretty much useless alien tech. And you needed 3 ayy items which can boost, to unlock everything ayy related.

Yeah if i checked git right, destructive analyser is only used for unlocking aylien tech and reclaiming some mats from an item.

**Comment by Carbonhell** [comment]
I'd rather prefer than the DA was integrated into R&D as an useful machinery once again, rather than this. -1

**Comment by Carbonhell** [comment]
Closing for majority of -1s

**Comment by xTrainx** [comment]
ok, i only did it to see Hippie stand on this, i will take the comments made in this pr to heart
